leadiv/notification-banner-wc
# feature: setup ci

## What's New?
This sets up the basic continuous integration for all branches.

## How to test?
1. Pull done the branch and run `npm install`
2. Run `npm test`
3. Run `npm serve`
4. Should all work as expected.
